docs/configuration.md

```diff
@@ -90,6 +90,7 @@ Aside from the comments in YAML file itself the fields are described below.
 | `dnatracing` | `run`                          | boolean    | `true`         | Whether to run DNA Tracing.  Options : true, false                                                                                                                                                                                                           |
 | `plotting`   | `run`                          | boolean    | `true`         | Whether to run plotting. Options : `true`, `false`                                                                                                                                                                                                           |
 |              | `save_format`                  | string     | `png`          | Format to save images in, see [matplotlib.pyplot.savefig](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.savefig.html)                                                                                                                          |
+|              | `pixel_interpolation`          | string or None  | `None`    | Interpolation method for matplotlib image plots (when one or both of `axes` or `colorbar` is true). See [matplotlib imshow intepolations](https://matplotlib.org/stable/gallery/images_contours_and_fields/interpolation_methods.html).                                                                                                                         |
```

Comment:
Ah, you pushed this whilst I was writing my review.

```suggestion
|              | `pixel_interpolation`          | string or None  | `None`    | Interpolation method for matplotlib image plots (when one or both of `axes` or `colorbar` is true). The recommended default of `None` prevents banding that occurs in high resolution images when `nearest` is used. If interpolation is required it is suggested to use `gaussian`. For all possible options refer to the [matplotlib imshow interpolations documentation](https://matplotlib.org/stable/gallery/images_contours_and_fields/interpolation_methods.html).                                                                                                                         |
```

Reply:
I like this better than what I put as you were writing this! I'll update it now.